source/connect/connection-options/cluster-settings.txt

```diff
@@ -0,0 +1,188 @@
+.. _golang-cluster-settings:
+
+==========================
+Customize Cluster Settings
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, read preference, write
```

Comment:
S: consider making the keywords more specific to clusters/connections, since I don't think read preferences are mentioned